packages/apps/transfer/src/pages/transfer/cross-chain-transfer-tracker/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { styled } from '@kadena/react-components';
+
+export const StyledMainContent = styled('main', {
+  display: 'flex',
+  justifyContent: 'flex-start',
+  gap: '$16',
+});
+export const StyledFormButton = styled('div', {
+  marginTop: '16px',
+  // display: 'flex',
+  // padding: '8px 16px',
+  alignItems: 'center',
+  gap: '$8',
+});
+export const StyledForm = styled('form', {
+  width: '75%',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-end',
+});
+export const StyledSideContent = styled('div', {
+  display: 'flex',
+  flexDirection: 'column',
+});
+export const StyledAccountForm = styled('div', {
+  padding: '16px',
+  background: 'rgba(71, 79, 82, 0.4)',
```

Comment:
do we have this color in stitches themes? if yes it's better to use predefined variables

Reply:
I don't think so, since I also saw this value on the `cross-chain-transfer` page as well as the global component `NavLink`. When applying the new design we might change all these values to the predefined vars